Sources/PackageModel/SwiftSDKBundle.swift

```diff
@@ -428,7 +428,18 @@ extension [SwiftSDKBundle] {
         for bundle in self {
             for (artifactID, variants) in bundle.artifacts {
                 for variant in variants {
-                    guard variant.metadata.supportedTriples.contains(hostTriple) else {
+                    guard variant.metadata.supportedTriples.contains(where: { variantTriple in
+                        if
+                            hostTriple.arch == variantTriple.arch &&
+                            hostTriple.vendor == variantTriple.vendor &&
+                            hostTriple.os == variantTriple.os &&
+                            hostTriple.environment == variantTriple.environment
+                        {
+                            return hostTriple.osVersion >= variantTriple.osVersion
+                        } else {
+                            return false
+                        }
```

Comment:
I've inlined this check just in `SwiftSDKBundle.swift` and rewrote it so that it doesn't depend on vendored `Triple` API and can be easily backported to 5.9.

Reply:
Seems fine for 5.9, but we shouldn't land it like this in `main` IMO

Reply:
WDYT of the previous `isRuntimeCompatible` name for a function to factor this into?

Reply:
This is the naming I'm using after all in the latest commit.